AnaMunoz-Fatima-Functions-0826.js

```diff
@@ -0,0 +1,132 @@
+//============================================
+// PART 2 
+//============================================
+
+
+// # 1 Calculate the Cube
+//============================================
+const calculateCube = (x) => {
+  return x**3;
+}
+console.log(calculateCube(5));
+
+
+// #2 Is a Vowel?
+//============================================
+const isAVowel = (letter) => {
+  if (letter === 'a') {
+  return true
+} 
+else {
+return false
+}
+}
+console.log(isAVowel("a"));
+// console.log(isAVowel("b"));//Test
+
+// // #3 Get Two Lenghts?
+//============================================
+
+const getTwoLengths = (word1,word2) => {
+  return [word1,word2].map((x) => x.length); //The map() method calls the provided function once for each element in an array, in order
+} 
+console.log(getTwoLengths("Hank", "Hippopopalous"));
+//
+
+
+// 4 Get Multiplet Lengths
+//============================================
+function getMultipleLengths(strArray)//A StrArray (String Array) stores an array of character strings.  
+{
+  const newArray = [];
+  for (let i=0; i <strArray.length; i++) {
+      newArray.push(strArray[i].length);
+  }
+  return newArray;
+}
+
+console.log(getMultipleLengths(["hello", "what", "is", "up", "dude"]));
+
+//============================================
+// PART 2 
+//============================================
+
+// 1 Maximum of Three Numbers
+//============================================
+
+function maxOfThree(num1, num2, num3) 
+{
+  var largest = Math.max(num1, num2, num3);
+  return largest;
+}
+
+console.log(maxOfThree(6, 9, 1));
+
+// 2 print the longest word 
+//============================================
+printLongestWord = (arr) => {
+  arr.sort((a,b) => b.length - a.length)
+  return arr[0]
+}
```

Comment:
again, we hadn't learned `sort()` yet; try doing some logic without using a fancy method. I think you can do it!